pkg/controller/v1beta1/inferenceservice/reconcilers/service/service_reconciler.go

```diff
@@ -37,27 +37,29 @@ var log = logf.Log.WithName("ServiceReconciler")
 
 // ServiceReconciler is the struct of Raw K8S Object
 type ServiceReconciler struct {
-	client       client.Client
-	scheme       *runtime.Scheme
-	Service      *corev1.Service
-	componentExt *v1beta1.ComponentExtensionSpec
+	client        client.Client
+	scheme        *runtime.Scheme
+	Service       *corev1.Service
+	componentExt  *v1beta1.ComponentExtensionSpec
+	ServiceConfig *v1beta1.ServiceConfig
 }
 
 func NewServiceReconciler(client client.Client,
 	scheme *runtime.Scheme,
 	componentMeta metav1.ObjectMeta,
 	componentExt *v1beta1.ComponentExtensionSpec,
-	podSpec *corev1.PodSpec) *ServiceReconciler {
+	podSpec *corev1.PodSpec,
+	serviceConfig *v1beta1.ServiceConfig, raw bool) *ServiceReconciler {
 	return &ServiceReconciler{
 		client:       client,
 		scheme:       scheme,
-		Service:      createService(componentMeta, componentExt, podSpec),
+		Service:      createService(componentMeta, componentExt, podSpec, serviceConfig, raw),
 		componentExt: componentExt,
 	}
 }
 
 func createService(componentMeta metav1.ObjectMeta, componentExt *v1beta1.ComponentExtensionSpec,
-	podSpec *corev1.PodSpec) *corev1.Service {
+	podSpec *corev1.PodSpec, serviceConfig *v1beta1.ServiceConfig, raw bool) *corev1.Service {
```

Comment:
```suggestion
	podSpec *corev1.PodSpec, serviceConfig *v1beta1.ServiceConfig, rawDeploymentMode bool) *corev1.Service {
```

Reply:
As it is not needed anymore (as per as the last update)\, I forgot to revert it in my last update, will do it tomorrow morning.

Reply:
removed the `raw` parameter.